listenbrainz_spark/fresh_releases/fresh_releases.py

```diff
@@ -14,7 +14,7 @@
 USERS_PER_MESSAGE = 5
 
 
-FRESH_RELEASES_ENDPOINT = "https://api.listenbrainz.org/1/explore/fresh-releases/"
+FRESH_RELEASES_ENDPOINT = "https://test-api.listenbrainz.org/1/explore/fresh-releases/"
```

Comment:
Is this still desired?

Reply:
Nope, thanks for pointing out. Will fix.